display_information/control-annotation-sublayer-visibility/README.md

```diff
@@ -0,0 +1,36 @@
+# Control annotation sublayer visibility
+
+Use annotation sublayers to gain finer control of annotation layer subtypes.
+
+![Control annotation sublayer visibility sample](ControlAnnotationSublayerVisibility.png)
+
+## Use case
+
+Annotation, which differs from labels by having a fixed place and size, is typically only relevant at particular scales. Annotation sublayers allow for finer control of annotation by allowing properties (e.g. visibility in the map and legend) to be set and others to be read (e.g. name) on subtypes of an annotation layer.
+
+An annotation dataset which marks valves as "Opened" or "Closed", might be set to display the "Closed" valves over a broader range of scales than the "Opened" valves, if the "Closed" data is considered more relevant by the map's author. Regardless, the user can be given a manual option to set visibility of annotation sublayers on and off, if required.
```

Comment:
This use case makes it sound like this is more about min/max scale than sublayer visibility. While I understand the sample demonstrates both, we have named it "... sublayer visibility". Can we simplify to:

"An annotation layer for water utilities may contain sublayers for water mains, pipes, and valves. One might turn off the visibility of water main and pipe sublayers to make the valves easier to identify."

I also have no idea what kinds of annotation sublayers would actually exist in this scenario. So might be worth checking what I wrote makes sense.

Reply:
That works perfectly, great suggestion!